src/compton/Compton.cc

```diff
@@ -57,10 +57,21 @@ Compton::Compton(const std::string &filehandle) {
  *
  * \param filehandle The name of the pointwise lib to build MG data from
  * \param grp_bds    A vector containing the multigroup bounds (in keV)
+ * \param opac_type  The type of opacity to build. Valid options for CSK v0.2
+ *                   are "jayenne" (for IMC-style opacities) or "capsaicin"
+ *                   (for Sn-style opacities). Any other string will cause CSK 
+ *                   to throw an exception 
+ * \param wt_func    The frequency weighting function used to numerically 
+ *                   integrate the opacities. Valid options for CSK v0.2 are
+ *                   "flat", "wien" or "planck." Any other string will cause 
+ *                   CSK to throw an exception.
+ * \param induced    Bool to toggle consideration of induced effects off/on
  * \param n_xi       The number of angular points/Legendre moments desired
  */
 Compton::Compton(const std::string &filehandle,
-                 const std::vector<double> &grp_bds, const size_t nxi) {
+                 const std::vector<double> &grp_bds,
+                 const std::string opac_type, const std::string wt_func,
```

Comment:
I think `opac_type` and `wt_func` are of input only.  As strings, they should be of type `const &` to avoid making a copy (an allocation).